support-frontend/assets/pages/contributions-landing/components/PaymentMethodSelector.tsx

```diff
@@ -112,231 +104,277 @@ const mapDispatchToProps = {
+							fullExistingPaymentMethods.map(
+								(
+									existingPaymentMethod: RecentlySignedInExistingPaymentMethod,
+								) => (
+									<>
+										<Radio
+											id={`paymentMethodSelector-existing${existingPaymentMethod.billingAccountId}`}
+											name="paymentMethodSelector"
+											type="radio"
+											value={existingPaymentMethod.paymentType}
+											onChange={() => {
+												props.updatePaymentMethod(
+													mapExistingPaymentMethodToPaymentMethod(
+														existingPaymentMethod,
+													),
+												);
+												props.updateSelectedExistingPaymentMethod(
 													existingPaymentMethod,
+												);
+											}}
+											checked={
+												props.paymentMethod ===
+													mapExistingPaymentMethodToPaymentMethod(
+														existingPaymentMethod,
+													) &&
+												props.existingPaymentMethod === existingPaymentMethod
+											}
+											arial-labelledby="payment_method"
+											label={
+												<PaymentMethodLabel
+													label={getExistingPaymentMethodLabel(
+														existingPaymentMethod,
+													)}
+													logo={
+														<PaymentMethodLogo
+															paymentMethod={mapExistingPaymentMethodToPaymentMethod(
+																existingPaymentMethod,
+															)}
+														/>
+													}
+													isChecked={
+														props.existingPaymentMethod ===
+														existingPaymentMethod
+													}
+												/>
+											}
+										/>
+
+										<div css={styles.explainerListContainer}>
+											Used for your{' '}
+											{subscriptionsToExplainerList(
+												existingPaymentMethod.subscriptions.map(
+													subscriptionToExplainerPart,
 												),
-											);
-											props.updateSelectedExistingPaymentMethod(
-												existingPaymentMethod,
-											);
-										}}
-										checked={
-											props.paymentMethod ===
-												mapExistingPaymentMethodToPaymentMethod(
-													existingPaymentMethod,
-												) &&
-											props.existingPaymentMethod === existingPaymentMethod
-										}
-										arial-labelledby="payment_method"
-										label={renderExistingLabelAndLogo(existingPaymentMethod)}
-										cssOverrides={radioCss}
+											)}
+										</div>
+									</>
+								),
+							)}
+
+						{paymentMethods.map((paymentMethod) => (
+							<Radio
+								id={`paymentMethodSelector-${paymentMethod}`}
+								name="paymentMethodSelector"
+								type="radio"
+								value={paymentMethod}
+								onChange={() => {
+									onPaymentMethodUpdate(paymentMethod);
+								}}
+								checked={props.paymentMethod === paymentMethod}
+								label={
+									<PaymentMethodLabel
+										label={getPaymentLabel(paymentMethod)}
+										logo={<PaymentMethodLogo paymentMethod={paymentMethod} />}
+										isChecked={props.paymentMethod === paymentMethod}
 									/>
```

Comment:
Again here's the actual change - using the new component instead of `renderLabelAndLogo`